komga-webui/src/locales/en.json

```diff
@@ -164,12 +164,12 @@
   },
   "browse_readlist": {
     "edit_elements": "Edit elements",
-    "edit_readlist": "Edit read list"
+    "edit_readlist": "Edit reading list"
   },
   "browse_series": {
-    "earliest_year_from_release_dates": "This is the earliest year from the release dates from all books in the series",
-    "series_no_summary": "This series has no summary, so we picked one for you!",
```

Comment:
There is no "we". The software is not sentient.

Reply:
dubious. Many apps use this kind of sentences to make the user feel like he is interacting with more than a machine.

Reply:
The number of style-guides or software with a team of non-programmers in charge of language doing it is close to zero.
The psychology clearly favours non-sentience.
If you want to really make it seem like you are shouting at the user, and introduce even more of this sentiment in other languages, then by all means leave in the exclamation mark at the end too.